tests/dynamic_api_test.py

```diff
@@ -790,6 +790,15 @@ def test_jit_of_broadcast2(self):
     x = jax.jit(lambda n: jnp.ones(2 * n))(3)
     self.assertAllClose(x, jnp.ones(2 * 3))
 
+  @unittest.skipIf(jtu.device_under_test() != 'iree', "iree test")
```

Comment:
Question about testing -
1) You want to take this out before merging as it duplicates a later test 
https://github.com/google/jax/blob/fa46af405132e055be0c52d8f3a54d9ac597af63/tests/dynamic_api_test.py#LL667-L673C41
```python
  @unittest.skipIf(jtu.device_under_test() != 'iree', 'iree test')
  def test_transpose(self):
    @partial(jax.jit, abstracted_axes=({0: 'h', 1: 'w'},))
    def f(x):  # f32[h, w] -> f32[w, h]
      return x.T


    f(np.ones((3, 5), dtype=np.float32))
```


or 
2) naming it and putting it in tests separately logs the issue for perpetuity and that's a good thing?

Reply:
Oops, didn't realize it was a duplicate! I guess we're just not running IREE under CI...

I'll probably remove it.

Reply:
Thank you!